common/persistence/visibility/store/query/util_test.go

```diff
@@ -0,0 +1,63 @@
+package query
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseExecutionDurationStr(t *testing.T) {
+	s := assert.New(t)
+
+	testCases := []struct {
+		input         string
+		expectedValue time.Duration
+		expectedErr   error
+	}{
+		{
+			input:         "123",
+			expectedValue: time.Duration(123),
```

Comment:
I think users will expect this to be seconds or maybe even millis, nanos is unlikely IMHO.

Reply:
It was the previous behavior, so I'm just reverting to that.

Reply:
Got it. Unfortunate oversight...